wls-auth-service/src/main/resources/db/migrations/h2/V10_0_addAdminPermissionsGetKonfigurierteWahltage.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO Permission
+VALUES ('00000000-0000-0009-0000-000000000001', 'Admin_BUSINESSACTION_GetKonfigurierteWahltage');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0009-0000-000000000001');
```

Comment:
```suggestion
VALUES ('00000000-0000-0000-0002-000000000001', '00000000-0000-0010-0000-000000000001');
```
auch die linke Spalte ist m.E. zu korrigieren. Deine Authority ID entspricht "WLS_WAHLVORSTAND" aber wenn man im Alten Code guckt wer "Admin_BUSINESSACTION_GetKonfigurierteWahltage" machen durfte, das war nur die Rolle/Authority ADMIN, bei uns aktuell mit Authority id : 00000000-0000-0000-0002-000000000001